lookout/style/format/visualizer/server.py

```diff
@@ -106,7 +106,7 @@ def return_features() -> Response:
                     "ground_truths": y.tolist(),
                     "predictions": y_pred.tolist(),
                     "sibling_indices": sibling_indices,
-                    "rules": describe_rules(rules.rules, fe),
+                    "rules": describe_rules(rules, fe),
```

Comment:
this needs to be changed back to `rules.rules` now that `predict` returns a `Rules` and not `Sequence[Rule]`

Reply:
just noticed Vadim also made a comment on that line, sorry for the duplicate